webshopping/lolshopping/views.py

```diff
@@ -43,9 +43,17 @@ def registerUserpage(request):
             #print(type(raw_password)) print out the type, str
             #account = authenticate(email=email, password=raw_password)
             #print(type(user)) #print out "<class 'lolshopping.models.Account'>"
-            #print(account) #print out the email address that the user insert in 
+            #print(account) #print out the email address that the user insert in
             #print(type(account)) #print out "<class 'lolshopping.models.Account'>"
+
             user_from_usermodel = UserModel._default_manager.get_by_natural_key(email)
+            # print(user_from_usermodel.new_salt) get the salt from db
```

Comment:
These print statements should be removed.

Reply:
sure do